pkg/apb/watch_pod.go

```diff
@@ -36,6 +36,22 @@ var (
 
 type updateDescriptionFn func(string)
 
+// ErrorCustomMsg - custom termination msg from pod's 'terminationMessagePath'
```

Comment:
>custom termination msg from pod's 'terminationMessagePath'

I think that this is generic enough that it could be used for any type of custom error message. What do you think of changing this to just say  `An error to propagate the custom error message to the callers`

Reply:
I will update